src/woo_publications/api/openapi.yaml

```
@@ -2,7 +2,55 @@ openapi: 3.0.3
 info:
   title: WOO Publications
   version: 0.1.0
-  description: WIP
+  description: |2
+
+    ## Documentatie
+
+    De Woo-Publicatiebank API stelt client-applicaties in staat om publicaties van documenten te raadplegen en te beheren in het kader van een open overheid. Deze API-documentatie bevat drie logische 'domeinen'.
```

```suggestion
    De Woo-Publicatiebank API stelt client-applicaties instaat om publicaties van documenten te raadplegen en te beheren in het kader van een open overheid. Deze API-documentatie bevat drie logische 'domeinen'.
```
Volgens mij is instaat een woord.

---

helaas, terug naar school! :stuck_out_tongue: